web/src/components/event-cards.js

```diff
@@ -104,17 +124,16 @@ class EventCards extends Component {
   render() {
     const {classes} = this.props;
     const cards = [];
-
     for (let cardIndex = 0; cardIndex < this.props.events.length; cardIndex++) {
-      if (cardIndex % 3 === 0) {
-        cards.push([]);
-      }
+      // if (cardIndex % 3 === 0) {
+      //   cards.push([]);
+      // }
       // Push each card component in cards
-      cards[cards.length - 1].push(
-        <Grid item xs={12} sm={6} md={4}>
+      cards.push(
+        <Grid item xs={12} sm={6} md={4} key={this.props.events[cardIndex].id}>
           <Card className={classes.card}
-                key={this.props.events[cardIndex].id}
                 value={this.props.events[cardIndex].id}
+                key={this.props.events[cardIndex].id}
```

Comment:
Add space back and front of '='

Reply:
I think that spaces in sides of '=' are not necessary. is that google style?